charts/api/templates/deployment-app-backend.yaml

```diff
@@ -83,6 +83,19 @@ spec:
 
 {{ include "api.commonAppEnvVars" . | indent 10 }}
 
+          {{- if $.Values.app.elasticSearchSharedIndexHost }}
```

Comment:
Instead of repeating these in all deployments, this should go into `api.commonAppEnvVars` which is defined in `_helpers.tpl`